src/main/kotlin/com/cjbooms/fabrikt/generators/controller/ControllerGeneratorUtils.kt

```diff
@@ -48,4 +49,41 @@ object ControllerGeneratorUtils {
 
     private fun httpVerbMethodName(verb: String, isSingleResource: Boolean) =
         if (isSingleResource) "${verb}ById" else verb
+
+
+    /**
+     * Enum definition for different cases of authentication
+     *
+     * N0_SECURITY when no @Secured Annotation is to be added neither an authentication input parameter
```

Comment:
**praise:** How did you manage to write this `NO` with a zero instead of an O? 😄

Reply:
skillz!